guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md

```diff
@@ -174,6 +175,10 @@ Sample of usage in layout:
 
 This would add a new column to the page layout.
 
+### block vs container {#block_vs_container}
+
+A block represents the end of the chain in rendering HTML for Magento. Containers contain blocks and can wrap them in an HTML tag. Also, containers will not render any output if there are no children assigned to them.
```

Comment:
I feel like this might be better as a bulleted list. Maybe something like:
```suggestion
* Blocks represents the end of the chain in rendering HTML for Magento.
* Containers contain blocks and can wrap them in an HTML tag.
* Containers do not render any output if there are no children assigned to them.
```

Reply:
Question: we don't mention children in this topic. Are 'children' 'blocks' in this context?